scripts/changeset/update-changelog.mts

```diff
@@ -13,6 +14,34 @@ function sleep(time: number) {
   });
 }
 
+async function getChangesetPr(ghRepo: string, retried = false) {
+  const octokit = new Octokit();
```

Comment:
Not sure why this was skipped at first. 🤔

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1972/files#diff-b5dbae1aaeb1da81c601f9a810383c55cd3fd847a69ef4567294bd82be355334R61

Do we not get the client via `@actions/github`?

Reply:
Good catch! https://github.com/FuelLabs/fuels-ts/pull/1972/commits/3c9a1a008b6a723f2be50e01eeac80aa8a4c116d